### DISCUSSION TITLE
Is there a way to reset SSI without removing Group data?

### DISCUSSION THREAD

**Discussion by Verzacliche** [Pryaxis/TShock#2234] [General]
As what the title says, please help me, I made a lot of groups already but I wanted to restart my SSIs, I don't know anything about coding btw.

**Comment by bartico6** [comment]
Yes, you need to remove the tsCharacter table from tShock's database.
If you're using sqlite (which is default), use [db browser for sqlite](https://sqlitebrowser.org/dl/) to open tshock.sqlite, or use whatever native management tool you need if you're using a different storage engine.

**Reply by Verzacliche** [reply]
Thanks! I got it all working now
